### DISCUSSION TITLE
Testing React Native components in AVA 5?

### DISCUSSION THREAD

**Discussion by BenTracyDotCom** [avajs/ava#3210] [Q&A]
Is there a way to test React Native components in AVA 5? All the docs I could find point to AVA 3-4 for any kind of React component testing and every method I try to render the component and check that it is rendered gives a "SyntaxError: Unexpected token '<' ".

**Comment by novemberborn** [comment]
There is no more direct Babel integration, so you'll have to precompile and either use `@babel/register` (IIRC) or perhaps go the TypeScript route.

React support in the `deepEqual` and `snapshot` assertions was also removed. Under the hood that system is extensible but it's not exposed yet.
